src/components/ElectronRedirect.tsx

```diff
@@ -1,5 +1,5 @@
 import React, { Component } from "react";
-import { Redirect } from "../utilities/routing";
+import { Redirect } from "react-router";
```

Comment:
Is there a direct reason for this? Cause this means switching form the react-native-dom router to the plain react one.